lib/timeline/timeline.dart

```diff
@@ -81,13 +82,10 @@ class TimelineScreen extends Screen {
         ServiceExtensionButton(extensions.togglePlatformMode).button;
 
     mainDiv.add(<CoreElement>[
-      createLiveChartArea(),
-      div(c: 'section'),
       div(c: 'section')
         ..layoutHorizontal()
         ..add(<CoreElement>[
-          pauseButton,
-          resumeButton,
+          createHotReloadButton(),
```

Comment:
Do we think users will need to reload often from this page? Is there a better place to expose this functionality? Or, do we expect to expose it from multiple pages?

Reply:
And - perhaps a todo for later - we probably only want to expose the hot reload button if we determine that the device we're debugging can be hot reloaded (that it support the flutter_tools reload functionality).

Reply:
we have the information to only expose the hot reload button if available as we get an event from the VM when each service call is registered. it would be nice to have a matching api for this as the one we have for service extensions.

Reply:
Honestly, I see more of a use for Hot Restart on this page (implementing this is on my todo list). Perhaps if your app is slow on start up or if there is a janky animation that is kicked off by app start, having a hot restart button here could be useful to kick off another app start.

We should probably re-visit where the best place to expose hot reload/hot restart is in general. I'm happy with moving the button wherever it fits best.

Reply:
Added logic to hide the reload button if it is not supported by the device.

Reply:
Exposing hot reload and restart buttons on multiple pages sounds good to me particularly if we can detect that the user did not launch the app from VSCode or IntelliJ.
For example, hot reload and restart buttons would also be nice in the debugger and inspector pages for this kind of users.
Example inspector use case:
Notice the widget tree is surprising so I try issuing a hot restart to see if the tree matches my expectations more.

Reply:
Devon and I discussed offline - we think we should reconvene once the inspector is landed and come up with a common UI for shared buttons/functionality. A title bar across the top with reload, restart, and service extension toggles perhaps?